README.md

```diff
@@ -33,33 +33,45 @@ By the end of this capstone, your API should be working locally and ready for cl
 1. **Install the Dev Containers extension** in VS Code (if not already installed)
 2. **Reopen in container**: When VS Code detects the `.devcontainer` folder, click "Reopen in Container"
    - Or use Command Palette (`Cmd/Ctrl + Shift + P`): `Dev Containers: Reopen in Container`
-3. **Wait for setup**: The container will automatically install Python, dependencies, and configure your environment
+3. **Wait for setup**: The API container will automatically install Python, dependencies, and configure your environment.
+   The PostgreSQL Database container will also automatically be created.
 
-### 3. Start PostgreSQL Database
-
-1. **Start the database** (run this in the VS Code terminal inside the dev container):
+### 3. Verify the PostgreSQL Database Is Running
 
    ```bash
-   docker-compose up -d postgres
+   docker-compose ps
    ```
 
-2. **Verify it's running**:
+   You should see the postgres service running.
+
+### 4. Configure Your Environment (.env)
```

Comment:
They have to copy env-sample to .env before reopening the project in a dev container, please move this step up to right after 3. **Open in VS Code**: step.